src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_access_restriction_commands.py

```diff
@@ -158,14 +158,14 @@ def test_webapp_access_restriction_add_service_endpoint(self, resource_group):
             JMESPathCheck('[1].action', 'Deny')
         ])
 
-    @ResourceGroupPreparer(location=WINDOWS_ASP_LOCATION_WEBAPP)
+    @ResourceGroupPreparer(location='centralus')
     def test_webapp_access_restriction_remove(self, resource_group):
         self.kwargs.update({
             'app_name': self.create_random_name(prefix='cli-webapp-nwr', length=24),
             'plan_name': self.create_random_name(prefix='cli-plan-nwr', length=24)
         })
 
-        self.cmd('appservice plan create -g {rg} -n {plan_name}')
+        self.cmd('appservice plan create -g {rg} -n {plan_name} --debug')
```

Comment:
Why --debug? What's the effect? It seems not necessary.

Reply:
Removed.